src/hooks/useSubmit/useSubmit.ts

```diff
@@ -316,12 +315,24 @@ export const fuelSign = async ({
   return [payload, { params, sig }]
 }
 
-const chainsOfAddressWithDeployedContract = (address: `0x${string}`) =>
-  Promise.all(
+const chainsOfAddressWithDeployedContract = async (
+  address: `0x${string}`
+): Promise<Chain[]> => {
+  const LOCALSTORAGE_KEY = "chainsWithByteCode"
```

Comment:
Shouldn't this key include the address?

Reply:
Whoops, you're right! Fixed in 38ffbdaed2d4dbcec15ef6e38149abcaeadb541c